src/components/Books/AddBook.js

```diff
@@ -69,38 +70,47 @@ const AddNewBook = () => {
   }, [isTitleValid, isAuthorValid, title, author]);
 
   return (
-    <Card>
-      <h3>Add New Book</h3>
-      <form onSubmit={addBookHandler}>
-        <div className={!isTitleValid ? 'invalid' : ''}>
-          <p>{!isTitleValid && 'Book title should be 3 to 55 char long!'}</p>
+    <Card extraclass={classes.form_container}>
+      <h2>Add New Book</h2>
+      <form onSubmit={addBookHandler} className={classes.form}>
+        <div
+          className={`${classes.form_control} ${
```

Comment:
- [ ] Please in the design there is no border-top on the form section, it would be better to use:: after or::before pseudo-elements or `hr` tag to create this line in the design.
Zeplin 
![image](https://user-images.githubusercontent.com/74744787/222904788-4ab27134-4556-453d-b2ac-e63bce0babd8.png)
Your design 
![image](https://user-images.githubusercontent.com/74744787/222904879-cd486285-fbd9-4cbc-9151-690691077bfe.png)